lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -49,30 +48,34 @@ def __init__(self, *args, **kwargs) -> None:
         )
 
     def _on_install(self, _) -> None:
-        slurm.install()
-        self.unit.set_workload_version(slurm.version())
+        self._slurm_manager.ops.install()
+        self.unit.set_workload_version(self._slurm_manager.ops.version())
```

Comment:
We can drop the `ops` property here now since we changed how the manager object is constructed now:

```suggestion
        self._slurm_manager.install()
        self.unit.set_workload_version(self._slurm_manager.version())
```